src/core/Microsoft.PowerToys.Settings.UI/Strings/en-us/Resources.resw

```diff
@@ -768,4 +768,10 @@
   <data name="FileExplorerPreview_RebootRequired.Text" xml:space="preserve">
     <value>A reboot may be required for changes to these settings to take effect</value>
   </data>
-</root>
+  <data name="FancyZones_ExcludeApps_TextBoxControl.PlaceholderText" xml:space="preserve">
+    <value>Example: outlook.exe</value>

```

Comment:
Technically speaking, `outlook.exe` should not be in the resources file since it should not be localized.

Reply:
Agreed, but it's hard to split them, right? I was hoping PlaceholderText would accept Runs (like TextBlock does) but it's a simple string. Any recommendations?

Reply:
I think for this i'm ok keeping it.  We have product names in loc files.